### DISCUSSION TITLE
Use CSS modules without the `.module` suffix

### DISCUSSION THREAD

**Discussion by guidokessels** [fwouts/previewjs#39] [Ideas]
I just tried out React Preview and I really like what I see so far!

I had some issues getting the CSS to work. We use CSS modules and the CSS import in the TS file was empty. I had to rename the file from `MyComponent.scss` to `MyComponent.module.scss` for React Preview to treat it as a CSS module and populate the import.

Would it be possible to use CSS modules without the `.module` suffix?

Thanks.

**Comment by fwouts** [comment]
Hi @guidokessels, thanks for the feedback!

This is currently not possible, **however** I've just discovered https://github.com/Summer-andy/vite-plugin-transform-css-modules which I could easily add as a default plugin to React Preview. Let me try that out and come back to you (give me a few days!).

**Comment by fwouts** [comment]
Good news, this is now merged in React Preview. Coming to you later this week, once I've stabilised a few remaining bugs :)

**Comment by fwouts** [comment]
Hey @guidokessels, this is now live in v0.4.2 🙂

**Comment by guidokessels** [comment]
Hey, sorry for the late response -- I was on holiday :)

This seems to work great with the latest version. Thanks a lot!
